### PR TITLE
WIP - Updates to WebServicesAccess.md

### PR DESCRIPTION
**Pull request by smellems**
- [x] Add Issues and consideration about bandwidth and R&D solution
- [x] DLP and data classification tool
- [ ] ..

### CODE REVIEW

File: wip-tec/en/WebServicesAccess.md [hunk 1]
```diff
@@ -27,9 +41,27 @@ When access to collaboration tools and sites is restricted, instead of increasin
...
+
+#### Digital Worker Profiles
+
+If not all groups currently require or want more open access to the Internet, we can expect that more and more of them will in the future.
+Current branches that would benefit:
+
+- IITB
+- ...
+
+#### Awareness
```
Comment (remyb2canada) [inline wip-tec/en/WebServicesAccess.md]: Add another sub category: Perimeter Protection

Explore and evaluate the use of an electronic communication labelling tool. A tool that forces all electronic content to be categorized by classification and can clearly alerts the end user of such classification.  Such tool is to augment the DLP program by providing it with metadata description of the electronic content where, should the content being uploaded does not match the security of the site, it flags infractions for later investigation or just blocks them entirely.

File: wip-tec/en/WebServicesAccess.md [hunk 2]
```diff
@@ -27,9 +41,27 @@ When access to collaboration tools and sites is restricted, instead of increasin
...
+
 Instead of banning access to certain tools and sites, making access open by default and encouraging the secure use will result in risks being better controlled.
 Informing users of expected behaviours when using the Internet will help them to protect against potential confidentiality or privacy breaches and to comply with existing ESDC and GC codes of conduct and policies.
 
+This information is already available on the Intranet and routinely communicated to users through, popups, emails and mandatory trainings.
+
 ## Next Steps
```
Comment (remyb2canada) [inline wip-tec/en/WebServicesAccess.md]: Reword. Some of that stuff is outside of our mandate (e.g. training opportunities).
Should you approve, the IT Strategy team will:

1- Work with the Operations team to start grouping ESDC employees into work profiles
2- Work with IT Security and DSO to open up access to online services based on work profiles, starting with the lowest risks ones
3- Work with IT Security and DSO to identify the strategic objectives and priority actions from the Departmental Security Plan that needs to be addressed in order to open up access to online services
4- Research, evaluate, and recommend Open Source solutions that would satisfy the risk mitigation needs outline above

Comment (gabrielcossette) [reply wip-tec/en/WebServicesAccess.md]: As much as I would like to do 4., isn't more the mandate of the R&D team?

File: wip-tec/en/WebServicesAccess.md [hunk 3]
```diff
@@ -56,8 +88,7 @@ Should you approve, the Strategy team will work with Security and others at IITB
 - https://onedrive.live.com/
 - https://owncloud.org/ (not SaaS)
 - https://www.pcloud.com/
-- https://labs.play-with-docker.com/ (Websockets blocked)
-- https://riot.im
+- https://riot.im/app/
 - https://open.rocket.chat/
 - https://www.seafile.com/ (not SaaS)
```
Comment (gabrielcossette) [inline wip-tec/en/WebServicesAccess.md]: Seafile doesn't offer SaaS anymore, not sure it's relevant to list.

File: wip-tec/en/WebServicesAccess.md [hunk 4]
```diff
@@ -27,9 +41,27 @@ When access to collaboration tools and sites is restricted, instead of increasin
 Users will find a way around the blocks if it makes their life easier.
 Security is more than just locking things down; user experience must also be considered.
 
+#### Bandwidth
+
+The R&D team has been piloting a to route traffic to site that use a lot of bandwidth directly to the Internet.
```
[the remark this answers is not in the document]
Comment (gabrielcossette) [reply wip-tec/en/WebServicesAccess.md]: I'm curious about this PoC. As far as I know, CSE ITSG-22 forbids split-tunneling:

> **B.4.3.5 End-System Network Configuration Requirements**
[OZ-NC-123] End-Systems should maintain the integrity of Edge Interfaces with other GC Zones while connected to an OZ. Dual-homing, split-tunnelling, or other shared network paths with the Public Zone by OZ End-Systems should not be permitted.

Ref. https://cyber.gc.ca/en/guidance/annex-b-operations-zone-oz-baseline-security-requirements-itsg-22

Also, you could add that SSC is working on implementing direct connections to Canadian IXPs (Internet Exchange Points), which will help with the bandwidth issue (e.g. access to popular cloud providers).

File: wip-tec/en/WebServicesAccess.md [hunk 5]
```diff
@@ -12,9 +12,23 @@ The recent [Direction on Enabling Access to Web Services: Policy Implementation
...
+
+Bandwidth use is already high and can slow down the network during peaks. Open access to more Websites and services could risk making the network slower since it will require more bandwidth.
+
+#### Lower Productivity
+
+There is a perception that open access to online services and tools will lower productivity.
+
+#### Large Number of Employees
+
+ESDC is a large department, with over 27000 employees, and some categories of employees might not require the same access to online services as IT, policy and other groups.
```
[the remark this answers is not in the document]
Comment (smellems) [reply wip-tec/en/WebServicesAccess.md]: I put "Large Workforce With Various Work Profiles".  Not sure about it.  Large work profiles?  Their amount varies?  Was'nt sure either

File: wip-tec/en/WebServicesAccess.md [hunk 6]
```diff
@@ -12,9 +12,23 @@ The recent [Direction on Enabling Access to Web Services: Policy Implementation
 
 > Departments are to reconfigure their web filtering rules to be open by default to the Internet, except for websites that support non-acceptable activities or behaviours. See [Categories to be blocked](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32588#appA). All other categories, including social media and web-based collaboration or chat tools, are to be open by default.
 
-### Issue
+### Issues
 
-Sites that are classified as instant messaging, video conferencing, cloud storage and other online collaboration tools are blocked for ESDC employees on the network. This blacklist can never be complete and in some case seem to prefer certain solutions over others. See list of [Sites that are blocked and not blocked](#sites-that-are-blocked-and-not-blocked).
+Sites that are classified as instant messaging, email, cloud storage and other online collaboration tools are blocked for ESDC employees on the network.
+
+These categories are usually blocked for the following reasons:
```
[the remark this answers is not in the document]
Comment (smellems) [reply wip-tec/en/WebServicesAccess.md]: Yes! this commit also adds the risks of exposure of information

File: wip-tec/en/WebServicesAccess.md (hunk 2 above)
Comment (remyb2canada) [reply wip-tec/en/WebServicesAccess.md]: @gabrielcossette , yeah. We'll have to engage them.
Maybe reward as "Leverge the IITB Innovation Framework to research, evaluate,  and recommend...."?

File: wip-tec/en/WebServicesAccess.md [hunk 7]
```diff
@@ -39,20 +39,19 @@ It will support the digital transformation of how public servants work and serve
...
 
 A solution that forces classification of documents and emails as well as basic rules around the sharing of information through the network would allow our firewall to detect and prevent that information from getting out of the network.
 
 #### Second Open Network
 
-The R&D team has been piloting a to route traffic to site that use a lot of bandwidth directly to the Internet.
-This type of solution could mitigate the risk of making network performance worse and could even reduce the stress on the corporate network.
-It would also eliminate the need to block sites that are flagged for using too much bandwidth.
+The R&D team is working on a proof of concept to route internet traffic away from the internal corporate network.
+This type of solution would reduce the use of strain on the network.
```
Comment (remyb2canada) [inline wip-tec/en/WebServicesAccess.md]: "This type of solution would reduce strain on the corporate network"

**Pull request merged**